Makefile

```diff
@@ -110,13 +110,9 @@ example-sample-pipeline: ## Run the sample-pipeline example
 example-ibm-tutorial: ## Run the IBM pipeline example
 	bash examples/ibm-tutorial/ibm-tutorial.sh
 
-.PHONY: docs-setup
-docs-setup: ## Install the tool to build the documentation
-	bash docs/bootstrap.sh
```

Comment:
I think we can remove this script then.

Reply:
Doh, I had done that but VS code was sitting at a commit message editor. Gonna take me a bit to get used to these new prompts.